config/config.json

```diff
@@ -51,7 +51,8 @@
         "RestrictTeamInvite": "all",
         "RestrictPublicChannelManagement": "all",
         "RestrictPrivateChannelManagement": "all",
-        "UserStatusAwayTimeout": 300
+        "UserStatusAwayTimeout": 300,
+        "MaxChannelsPerTeam": 2111
```

Comment:
Set this line to `"MaxChannelsPerTeam": 2000` and remove all the other changes please